samples/RestoreSQLDB/Restore-CohesityRemoteMSSQLObject-Wrapper.ps1

```diff
@@ -0,0 +1,148 @@
+<#
+    Use the generic cmdlet to create an MSSQL remote restore task
+
+    Example usage:
+        Restore-CohesityRemoteMSSQLObject-Wrapper.ps1 -JobId 1234 -SqlHost x.x.x.x -SqlObjectName "MSSQLSERVER/database" -TargetHost y.y.y.y -CaptureTailLogs:$false -NewDatabaseName database_new -NewInstanceName SQL2016 -TargetDataFilesDirectory "C:\" -TargetLogFilesDirectory "C:\temp" -DbRestoreOverwritePolicy:$true
+#>
+[CmdletBinding(DefaultParameterSetName = "Default", SupportsShouldProcess = $True, ConfirmImpact = "High")]
```

Comment:
If we have entries such as JobId, show how can they invoke other cmdlets and get this piped in here, if we are auto discovering it , probably mention the behavior (latest snapshot vs non latest) etc.

Reply:
Done.